web-console/developer/advanced/advanced.md

```diff
@@ -12,6 +12,49 @@ When clicking on a specific project, Projects remains selected in the navigation
 
 ![Add project details](img/image2.png)
 
+## Project Access
+The Project Access page allows users to add or remove access to a project.
+
+![project access page](img/membership1.png)
+
+### Add and Remove Values
+* Clicking add access will add a row at the bottom of the list.
+* Clicking the remove icon on the right side of a row will remove the row.
+* After adding or removing access, users must click save at the bottom of the page to apply changes.
+* In the first section, users will be prompted to enter a name for the user.
+* In the second section, there are three options will chose from for selecting a role: admin, view, or edit.
+
+![add project access](img/membership3.png)
+
+![add role](img/membership5.png)
+Users are able to add access to the project.
+
+![add name](img/membership7.png)
+
+![save add](img/membership9.png)
+
+![updated user access page add](img/membership12.png)
+This is the updated Project Access page after one access has been added.
+
+![remove user](img/membership30.png)
+Users are able to remove access to the project.
+
+![save remove](img/membership40.png)
+
+![updated user access page remove](img/membership1.png)
+This is the updated Project Access page after one access has been removed.
+
+![remove all access](img/membership50.png)
+Users are able to remove all access from the project.
+
+![save remove all access](img/membership14.png)
+
+![updated remove all access](img/membership60.png)
+This is the updated Project Access page after all access has been removed.
+
```

Comment:
Would this sound better?
_This is the updated Project Acces page after removing access of all users from this project._